modelator/cli/__init__.py

```diff
@@ -147,6 +147,52 @@ def typecheck():
         raise typer.Exit(code=6)
 
 
+@app.command(
+    context_settings={"allow_extra_args": True, "ignore_unknown_options": True}
+)
+def simulate(
+    ctx: typer.Context,
+    model_path: Optional[str] = typer.Option(None, help="Path to the TLA+ model file."),
+    config_path: Optional[str] = typer.Option(
+        None, help="Path to TOML file with model and model checker configurations."
+    ),
+    init: Optional[str] = typer.Option(None, help="Model's init predicate."),
+    next: Optional[str] = typer.Option(None, help="Model's next predicate."),
+    constants: Optional[str] = typer.Option(
+        None,
+        help="Comma-separated list of constant definitions in the format 'name=value' (overwrites config file).",
+    ),
+    simulation_length: Optional[int] = typer.Option(
```

Comment:
I think this option's name is too long. As a user I would be bothered to write it. How about just `length` since we are already in the `simulate` command? The same for `num_simulations` below.